CargoHooks.php

```diff
@@ -261,6 +261,12 @@ public static function onPageSaveComplete(
 		RevisionRecord $revisionRecord,
 		EditResult $editResult
 	) {
+		// Fandom-start
+		if ( $wikiPage->isRedirect() ) {
```

Comment:
Looks good for move with redirect, had following code on my devbox to verify:
```
	public function onPageSaveComplete( $wikiPage, $user, $summary, $flags, $revisionRecord, $editResult ): void {
		var_dump("Page moved, isRedirect: ", $wikiPage->isRedirect());die();
}
```

Results for move with redirect:
```
Page moved, isRedirect: true
```

Results for move without redirect page creation - the hook doesn't trigger since no new page is created so the page was simply moved.

Is that fine?

Maybe the issue doesn't occur for move without redirect, but wanted to double-check since I can see that we are also using `onPageMoveComplete` in this file, which also does some SQL updates